src/main/scala/advancepassengerinfo/importer/DqApiFeed.scala

```diff
@@ -45,10 +49,16 @@ case class DqApiFeedImpl(fileNamesProvider: FileNames,
             case None =>
               log.info(s"$zipFileName could not be processed")
               metricsCollector.counter("api-dq-zip-failure", 1)
-          }
-          .map(_ => zipFileName)
-      }
-      .wireTap(_ => metricsCollector.counter("api-dq-zip-processed", 1))
+          }.recover {
```

Comment:
If we can put the chained calls, eg `.recover` & `.wireTap` and `.map` on their own lines then they'll match the indentation of the other chained calls and will be easier to read